src/fabric/src/fabric_view_changes.erl

```diff
@@ -98,13 +98,15 @@ keep_sending_changes(DbName, Args, Callback, Seqs, AccIn, Timeout, UpListen, T0)
     LastSeq = pack_seqs(NewSeqs),
     MaintenanceMode = config:get("couchdb", "maintenance_mode"),
     NewEpoch = get_changes_epoch() > erlang:get(changes_epoch),
+    PendingCount = pending_count(Offset),
     if
+        Dir == rev, is_integer(PendingCount), PendingCount =< 0;
```

Comment:
Minor update: I asserted that PendingCount is an integer. If we haven't heard from at least on worker it will be `null`.  That would have also worked as `null` sorts greater than `0` but it's better to be explicit probably.